bw2data/signals.py

```diff
@@ -0,0 +1,20 @@
+from blinker import signal
+
+
+project_changed = signal('bw2data.project_changed', doc="""
```

Comment:
Which other signals are we implementing?

Reply:
I would say every CRUD operation as well. Probably more in the future. I haven't found a way to specify types for the objects which will be passed to the signal catchers; we can do that in the docs, but type hinting has replaced the need for extensive docs (to some degree) for me, and I would love to have it here if possible...